src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCounter = 1;
+
+  const getSum = (num) => {
+    return (sum) => {
```

Comment:
Just consider better naming. For example, if the function is called `makeRobotAccountant`, it should probably return something named `robot`🙂. Also, since we're adding two equivalent numbers, then maybe you could name them something like `num1` and `num2` instead of `sum` and `num`.